File: Programmers/Level1/SumMatrix(jpt)/sumMatrix.py
def sumMatrix(A,B):
    answer = []
    for i in range(len(A)):
        row = []
        for j in range(len(A[0])):
            row.append(A[i][j]+B[i][j])
        answer.append(row)
    return answer
print(sumMatrix([[1,2], [2,3]], [[3,4],[5,6]]))

File: Programmers/Level1/SumMatrix(jptjs)/summatrix.js
function sumMatrix(A,B){
	var answer = Array();
  for(var i=0; i<A.length; ++i)
  {
    var temp = Array();
    for(var j=0; j<A[0].length; ++j)
    {
      temp.push(A[i][j] + B[i][j]);
    }
    answer.push(temp);
  }
	return answer;
}

// 아래는 테스트로 출력해 보기 위한 코드입니다.
console.log(sumMatrix([[1,2], [2,3]], [[3,4],[5,6]])) 